plugins/widget/plugin.js

```diff
@@ -2901,6 +3004,80 @@
 		} );
 	}
 
+	function handleStylesOnDowncast( widget, widgetElement ) {
+		var styledElement;
+
+		if ( widgetElement ) {
+			if ( widget.styleDefinition && widgetElement.name !== widget.styleDefinition.element ) {
+				styledElement = widgetElement.getFirst( widget.styleDefinition.element );
+			}
+			if ( !styledElement ) {
+				styledElement = widgetElement;
+			}
+
+			styledElement.attributes = styledElement.attributes || {};
```

Comment:
What is the purpose of this line?

Reply:
Some tests were failing without this, but after other changes to code it's redundant leftover. I'm removing that.